practice_projects/#Homework pt.py

```diff
@@ -0,0 +1,19 @@
+#Homework pt.1
+
+friend = ("Software is my friend!")
+print (friend)
+
+#Homework pt.2
+
```

Comment:
I don't see #3: Copy/paste the following code and fix it in your script so that it prints properly with no errors:

variable_1 = I would like this to be a string
print variable_1

Reply:
Missed that one. It'll be on there on the next push